fabtests/prov/efa/Makefile.include

```diff
@@ -32,7 +32,10 @@
 
 bin_PROGRAMS += prov/efa/src/fi_efa_rnr_read_cq_error \
 		prov/efa/src/fi_efa_rnr_queue_resend \
-		prov/efa/src/fi_efa_info_test
+		prov/efa/src/fi_efa_info_test \
```

Comment:
Backslash continuation isn't necessary here

```suggestion
		prov/efa/src/fi_efa_info_test
```

Reply:
I have no write access, so can't resolve this convo myself

Reply:
Yes that was actually causing a build failure. Fixed now.